assets/src/apps/authoring/components/PropertyEditor/schemas/screen.ts

```diff
@@ -174,6 +192,24 @@ export const transformScreenSchematoModel = (schema: any) => {
     trapStateScoreScheme: schema.trapStateScoreScheme,
     negativeScoreAllowed: schema.negativeScoreAllowed,
   };
+  if (currentSequence?.custom.isBank) {
+    modelData.bankShowCount = schema.bankShowCount;
+    modelData.bankEndTarget = schema.bankEndTarget;
+  }
+  return modelData;
 };
 
-export default screenSchema;
+export const getScreenSchema = (seq: SequenceEntry<SequenceEntryChild> | null) => {
+  if (seq?.custom.isBank) {
+    return {
+      type: schema.type,
+      properties: {
+        Size: schema.properties.Size,
+        palette: schema.properties.palette,
+        bankShowCount: { type: 'number', title: 'Randomly selects question(s) from the bank' },
```

Comment:
perhaps we should do the collapsible section for "Question Bank Settings" OR a non collapsible section could make sense as well, but I think it would be good to separate them visually